aqua/cool-retro-term/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           qmake5 1.0
+PortGroup           github 1.0
+
+github.setup        Swordfish90 cool-retro-term 1.1.1
+revision            0
+
+categories          aqua shells
+platforms           darwin
+license             {GPL-3 GPL-2}
```

Comment:
That's unusual. Normally we just say:

`license             GPL-2+
`

Or does the license really specify only GPL 2 or 3 but not higher?

Reply:
Thanks for pointing that out. I couldn't find an explicit "or higher" anywhere prominent, and there was a `gpl-2.0.txt` and `gpl-3.0.txt` in the repo root, hence `{GPL-3 GPL-2}`.

But then I found [this](https://github.com/Swordfish90/cool-retro-term/blob/1cf6e2743b86cc0b07c9042945c7c94c61c770e5/packaging/appdata/cool-retro-term.appdata.xml#L6) which indicates just `GPL-3+` should be correct, so I've fixed it.